### DISCUSSION TITLE
Condense my user and group name in long lists

### DISCUSSION THREAD

**Discussion by c256** [eza-community/eza#1265] [Development]
I patched my local install to replace the current user and group name to "~" in long listings, for brevity, giving me something like this:

> $ eza -lgF
> drwxr-xr-x - ~ ~  3 Dec 14:25 bin/
> drwxr-xr-x - ~ ~  9 Dec 23:50 CNotes/
> drwxr-xr-x - ~ ~ 29 Apr  2023 lib/
> drwxr-xr-x - ~ ~ 18 Jun  2021 LSG/
> lrwxrwxrwx - ~ ~ 11 Oct  2019 Project -> /usr/local/src/
> drwxr-xr-x - ~ ~ 30 Oct 13:32 tmp/

The change was quite small, but I haven't bothered to make it optional, update the tests, etc. Is there any interest in upstreaming this idea?

**Comment by cafkafk** [comment]
We already have `--no-users` maybe we should just add a `--no-groups`?

**Reply by c256** [reply]
I don't know if anyone else cares about hairs this finely split, but, to be clear, my patch effectively de-emphasizes my (thus default) user & group versus others.  For example:

> $ eza -lgF
> drwxr-xr-x - root root   2 Jan  2020 audacity/
> drwxrwxr-x - root staff 23 Aug 07:48 emacs/
> drwxrwxr-x - ~    staff 27 Nov 19:41 emacs-igc/
> drwxr-xr-x - root root  20 Sep  2019 google/
> drwxr-xr-x - ~    ~     21 Oct  2022 yoga/

I can totally believe that nobody else really wants this. 😄 

Thanks!
